develop/toolkit/blockchain/spawn-networks/zombienet/spawn-basic-network.md

```diff
@@ -0,0 +1,312 @@
+
+![](/images/develop/toolkit/blockchain/spawn-networks/zombienet/spawn-basic-network/spawn-basic-network-01.webp)
+
+### Check Logs
+
+To check node logs, you can use the command listed in the output as `Log Cmd`. For instance, to check the logs of the `alice` node, you can open a new terminal and run the following command:
+
+```bash
+tail -f /var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-SEzfCidQ1za4/alice.log
+```
+
+After running this command, you will see the logs of the `alice` node in real-time, which can be useful for debugging purposes. The logs of the `bob` and `collator01` nodes can be checked similarly.
+
+## Testing DSL
+
+Zombienet provides a Domain Specific Language (DSL) for writing tests. The DSL is designed to be human-readable and allows you to write tests using natural language expressions. You can define assertions and tests against the spawned network using this DSL. This way, users can evaluate different metrics, such as:
+
+- **On-chain storage** - the storage of each of the chains running via Zombienet
+- **Metrics** - the metrics provided by the nodes
+- **Histograms** - visual representations of metrics data
+- **Logs** - detailed records of system activities and events
+- **System events** - notifications of significant occurrences within the network
+- **Tracing** - detailed analysis of execution paths and operations
+- **Custom API calls (through Polkadot.js)** - personalized interfaces for interacting with the network
+- **Commands** - instructions or directives executed by the network
+
+These abstractions are expressed by sentences defined in a natural language style. Therefore, each test line will be mapped to a test to run. Also, the test file (`*.zndsl`) includes pre-defined header fields used to define information about the suite, such as network configuration and credentials location.
+
+## The Test File
+
+The test file is a text file with the extension `.zndsl`. It is divided into two parts: the header and the body. The header contains the network configuration and the credentials to use, while the body contains the tests to run.
+
+The header is defined by the following fields:
+
+- **`description`** ++"string"++ - long description of the test suite (optional)
+- **`network`** ++"string"++ - path to the network definition file, supported in both `.json` and `.toml` formats
+- **`creds`** ++"string"++ - credentials filename or path to use (available only with Kubernetes provider). Looks in the current directory or `$HOME/.kube/` if a filename is passed
+
+The body contains the tests to run. Each test is defined by a sentence in the DSL, which is mapped to a test to run. Each test line defines an assertion or a command to be executed against the spawned network.
```

Comment:
This basic information about testing should remain it's own page and not be nested into a tutorial. I'd prefer to see the content split up as we had it before in the ecosystem docs draft https://papermoonio.github.io/polkadot-ecosystem-docs-draft/dev-tools/zombienet/testing/
https://papermoonio.github.io/polkadot-ecosystem-docs-draft/dev-tools/zombienet/tutorials/spawn-a-basic-network/